src/components/productView/productView.js

```diff
@@ -185,45 +175,10 @@ const ProductView = ({ routeDetail, t, toolbarGraph, toolbarGraphDescription, to
 /**
  * Prop types.
  *
- * @type {{t: translate, toolbarGraph: (Node|boolean), toolbarGraphDescription: boolean, routeDetail: object,
- *    productConfig: object, toolbarProduct: (Node|boolean)}}
+ * @type {{t: translate, toolbarGraph: (Node|boolean), toolbarGraphDescription: boolean, productConfig: object,
+ *    toolbarProduct: (Node|boolean)}}
  */
 ProductView.propTypes = {
```

Comment:
Optional: since we're removing the propTypes description would be good to confirm the config tests check for these properties

Reply:
added productLabel check to existing config pass/fail